pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -24,11 +24,8 @@ import (
 )
 
 const (
-	appImgObj = "appImg.obj"
-	certObj   = "cert.obj"
-	agentName = "zedmanager"
-
-	certificateDirname = persistDir + "/certs"
+	agentName    = "zedmanager"
+	finalDirname = types.ImgCatalogDirname + "/verified"
```

Comment:
You've already define VerifiedDirname so please use it.

Reply:
Done..